b2sdk/transfer/inbound/downloaded_file.py

```diff
@@ -196,10 +240,34 @@ def save_to(self, path_, mode='wb+', allow_seeking=True):
         :param allow_seeking: if False, download strategies that rely on seeking to write data
                               (parallel strategies) will be discarded.
         """
+        path_ = pathlib.Path(path_)
+        is_stdout = points_to_stdout(path_)
+        if is_stdout or points_to_fifo(path_):
+            if mode not in (None, 'wb'):
+                raise DestinationError(f'invalid mode requested {mode!r} for FIFO file {path_!r}')
+
+            if is_stdout and _IS_WINDOWS:
+                if self.write_buffer_size and self.write_buffer_size not in (
+                    -1, io.DEFAULT_BUFFER_SIZE
+                ):
+                    logger.warning(
+                        'Unable to set arbitrary write_buffer_size for stdout on Windows'
+                    )
+                context = contextlib.nullcontext(sys.stdout.buffer)
+            else:
+                context = open(path_, 'wb', buffering=self.write_buffer_size or -1)
+
+            try:
+                with context as file:
+                    return self.save(file, allow_seeking=allow_seeking)
+            finally:
+                if not is_stdout:
+                    set_file_mtime(path_, self.download_version.mod_time_millis)
```

Comment:
Does this set mtime for fifo? Is ti possible?

Reply:
Weirdly enough, only Windows is complaining about these things

Reply:
```
$ ls -la test_fifo                                      9:59:51
prw-rw-r-- 1 rooter rooter 0 Nov 15 09:59 test_fifo
$ touch -t 10101010 test_fifo                           9:59:54
$ ls -la test_fifo                                      9:59:58
prw-rw-r-- 1 rooter rooter 0 Oct 10 10:10 test_fifo
```